lib/jsdom/living/nodes/CharacterData-impl.js

```diff
@@ -79,6 +80,7 @@ class CharacterDataImpl extends NodeImpl {
     }
 
     queueMutationRecord(MUTATION_TYPE.CHARACTER_DATA, this, null, null, this._data, [], [], null, null);
+    invalidateStyleCache(this);
```

Comment:
I'm not sure this is necessary? When could changing text data cause a style update?